scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A script to check that travis.yml file & protractor.conf.js have the
+same test suites."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+import python_utils
+import utils
+
+
+def extract_travis_jobs_section():
```

Comment:
This is just the e2e jobs, right, not all jobs?

Also the function should be named get_e2e_suite_names_from_travis_yml_file or similar. "extract travis job section" suggests that the entire section is being returned, which is not the case.

Ditto for other functions, be careful with naming. Try to be as precise as possible.

Reply:
Okay sorry, will give more focus semantic naming.
Done.